storage/innobase/xtrabackup/src/CMakeLists.txt

```diff
@@ -134,6 +134,11 @@ TARGET_LINK_LIBRARIES(xtrabackup
    ADD_INSTALL_RPATH_FOR_PROTOBUF(xtrabackup)
  ENDIF()
 
+ # KMIP depend on boost
+ INCLUDE_DIRECTORIES(SYSTEM
+        ${BOOST_PATCHES_DIR}
+        ${BOOST_INCLUDE_DIR})
+
```

Comment:
will this link boost to plugin or xtrabackup?

Reply:
```
marcelo@marce-bld:/work/pxb/src/8.0$ ldd /work/pxb/ins/8.0/bin/xtrabackup  | grep boost | wc -l
0
marcelo@marce-bld:/work/pxb/src/8.0$ ldd /work/pxb/ins/8.0/lib/plugin/component_keyring_kmip.so  | grep boost | wc -l
0
```
neither one.